server/src/main/java/org/elasticsearch/health/node/action/NotHealthNodeException.java

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+package org.elasticsearch.health.node.action;
+
+import org.elasticsearch.ElasticsearchException;
+import org.elasticsearch.common.io.stream.StreamInput;
+
+import java.io.IOException;
+
+/**
+ * Exception which indicates that an operation failed because the node stopped being the selected health node.
+ */
+public class NotHealthNodeException extends ElasticsearchException {
```

Comment:
I don't think we need this kind of exception ?

it doesn't seem used, is it?

Reply:
You are right, I missed it, it's from the previous set-up :). I will remove it.